i18n/strings.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/* THIS IS A GENERATED FILE. DO NOT EDIT DIRECTLY */
+$i18nStrings = array(
+	"Saving…" => __( "Saving…" ),
+	"Save Settings" => __( "Save Settings" ),
+	"Saving..." => __( "Saving..." ),
+	"Save changes" => __( "Save changes" ),
+	"Add package" => __( "Add package" ),
+	"Apply changes" => __( "Apply changes" ),
+	"Cancel" => __( "Cancel" ),
+	"View exterior dimensions" => __( "View exterior dimensions" ),
+	"Invalid value" => __( "Invalid value" ),
+	"This field is required" => __( "This field is required" ),
+	"Edit package" => __( "Edit package" ),
+	"Add a package" => __( "Add a package" ),
+	"Package name" => __( "Package name" ),
+	"The customer will see this during checkout" => __( "The customer will see this during checkout" ),
+	"This field must be unique" => __( "This field must be unique" ),
+	"Inner Dimensions (L x W x H) %s" => __( "Inner Dimensions (L x W x H) %s" ),
+	"Outer Dimensions (L x W x H) %s" => __( "Outer Dimensions (L x W x H) %s" ),
+	"Package weight" => __( "Package weight" ),
+	"Weight of box" => __( "Weight of box" ),
+	"Max weight" => __( "Max weight" ),
+	"Max weight" => __( "Max weight" ),
```

Comment:
Oops! Looks like this tool doesn't remove duplicates (I wonder if they are aware of this in Calypso). Great catch!

:construction_worker: _Implementing de-duplication_